docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.9'
 services:
   beyond:
-    build: .
```

Comment:
Why was the build key removed?

Reply:
Because the image build stage is now removed from this flow.
The image is ready and being hosted on quayei already. We just need to pull it and thats it.

Reply:
Something is wrong with this concept.  An automatic build should happen when a code change lands in master.

Reply:
I agree, there should be a different flow for building and pushing the image. lets discuss this in my reply on your comment from above (to avoid duplicated messages)